scipy-1.0/references.bib

```diff
@@ -1679,3 +1679,27 @@ @online{ckdtree-weights
 url = {https://github.com/scipy/scipy/pull/5647},
 urldate = {2019-1-28}
 }
+
+@online{scipy-gh2885,
+author = {{Pauli Virtanen}},
+title = {{ENH: interpolate: rewrite ppform evaluation in Cython}},
+year = 2013,
+url = {https://github.com/scipy/scipy/pull/2885},
+urldate = {2019-2-15}
+}
+
+@online{scipy-gh3174,
+author = {{Evgeni Burovski}},
+title = {{add b-splines}},
+year = 2013,
+url = {https://github.com/scipy/scipy/pull/3174},
+urldate = {2019-2-15}
+}
```

Comment:
I might be missing something, or might have missed some previous discussions (please point me to one if that's the case): why adding these as references, here and elsewhere? FWIW I don't think it's wanted or needed. A PR is not a single-person affair and is not worthy of an academic citation.

Reply:
I think there was a discussion in support of citing them, or maybe it was just a review comment from @mdhaber or @ilayn? We've also got mailing list messages and all sorts of barely-academic things cited. Some were set as footnotes, but footnotes are barred for the journal we will target first.

I don't really mind if they are removed, but then someone should open an issue, and get a reasonable number of people to make a decision. I'm not sure if the journal will allow those citations anyway.

Reply:
So the issue is whether to cite GitHub PRs/issues or not?
I don't remember the history but I might have said something in favor of them before.
If they are primary sources, I think the citations should at least be collected so we know where we are getting our facts from. 
Citing websites is often discouraged for good reason, but I think this is if the website is a potentially unreliable secondary (or _n_-ary) source. For what we are writing, websites are often the primary source. Citing primary sources is something that certainly makes sense in many disciplines.
So I would suggest collecting/keeping the citations and removing them if the journal asks us to.